examples/meta/generator/targets/python.json

```diff
@@ -81,7 +81,7 @@
             "get_string": "$object.get($arguments)"
         },
         "StaticCall": "sg.$typeName.$method($arguments)",
-        "GlobalCall": "sg.$method($arguments$kwargs)",
+        "GlobalCall": {"Default": "sg.shogun.$method($arguments)"},
```

Comment:
I am not sure I understand why you changed this?

Reply:
i change the examples/meta/generator/translate.py, so the"GlobalCall" should be a list.